site/cypress/e2e/Cypress-Autograding/development_courses.spec.js

```diff
@@ -0,0 +1,64 @@
+const checkGradeable = (gradeable_id) => {
+    cy.visit(['development', 'gradeable', gradeable_id]);
+    cy.get('[data-testid="new-submission-info"]').contains('New submission for');
```

Comment:
For this and other Cypress tests, its better to explicitly state `cy.get(x).should('contain.text', 'text')`. While cy.get().contains() technically works, it's not best practice, as it is meant to return the Cypress object, not to test existence.